src/main/java/de/tum/in/www1/artemis/repository/StatisticsRepository.java

```diff
@@ -0,0 +1,60 @@
+package de.tum.in.www1.artemis.repository;
+
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import de.tum.in.www1.artemis.domain.User;
+
+/**
+ * Spring Data JPA repository for the user statistics
+ */
+@Repository
+public interface StatisticsRepository extends JpaRepository<User, Long> {
+
+    @Query("select count(distinct u.login) from User u, PersistentAuditEvent p where u.login like p.principal and p.auditEventType = 'AUTHENTICATION_SUCCESS' and u.login not like '%test%' and p.auditEventDate >= :#{#span}")
```

Comment:
Could you please reformat the querys using Java 13 Textblocks? See: https://thorben-janssen.com/java-text-blocks-hibernate-jpa/
It makes them much nicer to read imo :) 
Concerns all querys

Reply:
Thanks for the hint, [done](https://github.com/ls1intum/Artemis/pull/2471/commits/ea28fa7ce583b5250cde3499d6659b38eb254195)